versioned_docs/version-2.1.0/06-concepts/11-authentication/01-setup.md

```diff
@@ -97,7 +97,7 @@ void run(List<String> args) async {
 | **emailSignInFailureResetTime** | The reset period for email sign in attempts. Defaults to 5 minutes.| 5min |
 | **userCanEditUserImage** | True if users can update their profile images.| true |
 | **userCanEditUserName** | True if users can edit their user names. | true |
-| **userCanEditFullName** | True if users can view their user name. | true | 
```

Comment:
Suggestion: Apply this change to the non versioned docs as well.

If you search for this line you can modify it in all places.

Reply:
Sure, on it.

Reply:
https://github.com/serverpod/serverpod_docs/pull/169

Reply:
Thanks for being so quick on this. Could you add that change as a commit to this PR instead? Then we have the full fix as one change in main later.

Reply:
Done. Please check.